Makefile

```diff
@@ -225,3 +225,29 @@ serve-docs:
 	-it -p 4000:4000 \
 	jekyll/jekyll \
 	jekyll serve --livereload
+
+# gen-docs generates a new versioned docs directory under site/docs. It follows
```

Comment:
I'm wondering if it's really worth duplicating this documentation both here and in the script, it'll be two places we'll need to update it if we change anything. That said, I'm also not sure where the best place to document this would be (probably here given it's unlikely that gen-docs.sh will be used directly?

Reply:
Yeah - I'm ok with just having it in the Makefile, I'll just mention in the script that it's used for a make target